forward_script/tests/PageInformationTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+
+class PageInformationTest extends PHPUnit_Framework_TestCase {
+
+	public function testMissingIdCanBeQueried() {
+		$pageInformation = new \Wikimedia\ForwardScript\PageInformation( (object)[
```

Comment:
You could simplify this by "using" `Wikimedia\ForwardScript\PageInformation` at the top of the file, as done in other tests.